packages/turf-rhumb-bearing/index.js

```diff
@@ -27,16 +27,16 @@ export default function (start, end, final) {
     if (!start) throw new Error('start point is required');
     if (!end) throw new Error('end point is required');
 
-    const coordsStart = getCoord(start);
-    const coordsEnd = getCoord(end);
-    const origin = new LatLon(coordsStart[1], coordsStart[0]);
-    const destination = new LatLon(coordsEnd[1], coordsEnd[0]);
-    let bear360;
+    var coordsStart = getCoord(start);
```

Comment:
@stebogit Let me see if we could compile our `index.js` from ES6 to ES5... (without too much hassle)